src/core_plugins/status_page/public/status_page.js

```diff
@@ -18,6 +18,7 @@
  */
 
 import 'ui/autoload/styles';
+import 'ui/i18n';
```

Comment:
This import registers the angular `i18nProvider`, which is seemly absent otherwise. It's normally included as part of an application's `ui/autoload`, but the status page only includes the `ui/autoload/styles` import. I thought it was safer to only include what was required, than to include all of the autoload. Without adding this import, the navbar fails to load, and the console logs an error:

![image](https://user-images.githubusercontent.com/3493255/47850437-9ee8c080-ddab-11e8-87c3-41e11dc2a81e.png)